task5/src/main/java/com/clouway/adapter/http/LoginPage.java

```diff
@@ -25,6 +25,7 @@ public LoginPage(SessionRepository sessionRepository) {
 
   protected void doGet(HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException {
     String errorMessage = request.getParameter("errorMsg");
+    sessionRepository.cleanExpired();
```

Comment:
This will be removed from here. I will leave it only in LoginFilter, so each time login page is requested, the inactive sessions will be deleted from the DB.